src/main/java/core/basesyntax/figures/Circle.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.Figure;
+
+public class Circle extends Figure {
+    private final int radius;
+    private final String name = "circle";
+
+    public Circle(String color, int radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public double getArea() {
+        return Math.PI * radius * radius;
+    }
+
+    @Override
+    public String toString() {
+        return "Figure: "
+                + name
```

Comment:
```suggestion
                + Circle.class.getSimpleName()
```
**Here and in the all other figures.**